components/camera/InfoScreenButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from 'expo-router'
+import { useTranslation } from 'react-i18next'
+
+import IconButton from '@/components/shared/IconButton'
+import { useOffenceStoreContext } from '@/state/OffenceStore/useOffenceStoreContext'
+
+export type TorchState = 'on' | 'off'
+
+export const InfoScreenButton = () => {
+  const { t } = useTranslation()
+  const generatedEcv = useOffenceStoreContext((state) => state.ecv)
+
+  return generatedEcv ? (
```

Comment:
(optional) Why such a strange naming? Why `generatedEcv` ? In the context of this button it is just `ecv`. You even pull it from a `state.ecv`. Why do you add more characters to type?

Moreover, it just makes a person pause, when reading this code without further (historical) context. "Why are we renaming the `ecv` to `generated`. Where is it generated from?" and so on... which I think just adds to mental load when reading this simple code.